pytket/package.md

```diff
@@ -35,10 +35,12 @@ For worked examples using TKET see our [notebook examples](https://tket.quantinu
 
 For bugs and feature requests we recommend creating an issue on the [github repository](https://github.com/CQCL/tket).
 
-User support: tket-support@cambridgequantum.com
+User support: tket-support@quantinuum.com
```

Comment:
We are still using the old address for support, because of slack integration.

Reply:
Will change it back for now.